README.md

```diff
@@ -1,2 +1,23 @@
-# HW01
-Edit the readme
+---
+title: "Homework 1"
+author: "Ben Wang"
+---
+
+### bwang5@uchicago.edu
+
+Hello all!
+My name is Ben, I'm an incoming first year student in the **University of Chicago** *Neurobiology* program [link here](https://neuroscience.uchicago.edu/). I am interested in studying the molecular mechanisms behind automatic functions such as breathing, pain and prioprioception. I have no prior experience in coding but have always been interested in learning!
```

Comment:
Hi Ben, my name it Ariana, an incoming student in the Computational Neuro program, really nice to e-meet you!  I'm excited to go through this course with you and hopefully get to know you better through the neuro program once the quarter begins.